opae.cfg

```diff
@@ -268,6 +268,126 @@
       }
     },
 
+    "n5013": {
+      "enabled": true,
+      "platform": "Silicom FPGA SmartNIC N5013",
+
+      "devices": [
+        { "name": "n5013_pf", "id": [ "0x1c2c", "0x1002", "0", "0" ] }
+      ],
+
+      "opae": {
+        "plugin": [
+          {
+            "enabled": true,
+            "module": "libxfpga.so",
+            "devices": [ "n5013_pf" ],
+            "configuration": {}
+          }
+        ],
+        "fpgainfo": [
+          {
+            "enabled": true,
+            "module": "libboard_n5010.so",
+            "devices": [
+              { "device": "n5013_pf", "feature_id": "0xe" },
+            ]
+          }
+        ],
+        "fpgad": [
+          {
+            "enabled": true,
+            "module": "libfpgad-vc.so",
+            "devices": [ "n5013_pf" ],
+            "configuration": {
+              "cool-down": 30,
+              "get-aer":     [ "setpci -s %s ECAP_AER+0x08.L",
+                               "setpci -s %s ECAP_AER+0x14.L" ],
+              "disable-aer": [ "setpci -s %s ECAP_AER+0x08.L=0xffffffff",
+                               "setpci -s %s ECAP_AER+0x14.L=0xffffffff" ],
+              "set-aer":     [ "setpci -s %s ECAP_AER+0x08.L=0x%08x",
+                               "setpci -s %s ECAP_AER+0x14.L=0x%08x" ],
+              "sensor-overrides": [],
+              "monitor-seu": true
```

Comment:
Does this device support Single Event Upset via the FME error registers?  Just checking to be sure, because as of Arrow Creek, SEU is detected via the flight data recorder.

Reply:
The N501x cards do not support the flight data recorder. But I checked with our FPGA team and it seemd the SEU bit is not implemented either in the old OFS version used. So I have changed the monitoring to false.

I had also made a json syntax error which I have fixed. So I did a forced push.